src/StravaSharp/ActivityClient.cs

```diff
@@ -206,5 +206,19 @@ public async Task<List<Stream>> GetActivityStreams(long activityId, params Strea
             var response = await _client.RestClient.Execute<List<Stream>>(request);
             return response.Data;
         }
+
+        /// <summary>
+        /// Summit Feature. Returns the zones of a given activity. Requires activity:read for Everyone and Followers activities. Requires activity:read_all for Only Me activities.
+        /// </summary>
+        /// <param name="activityId"></param>
+        /// <returns></returns>
+        public async Task<List<ActivityZone>> GetActivitieZones(long activityId)
```

Comment:
Nitpicker corner: method name should probably be `GetActivityZones`

Reply:
@rjvdboon  Good catch!